elements/pl-drawing/mechanicsObjects.js

```diff
@@ -771,26 +771,33 @@ mechanicsObjects.LatexText = fabric.util.createClass(fabric.Object, {
         let ref = this;
         var callback = function() {
             let svg = MathJax.tex2svg(str, {'display': false}).children[0];
-            
+
             /* SVG's generated by MathJax reference a list of sharedbase elements,
                so replace each reference with the actual element value. */
-            $(svg).find("use").each((_, use) => {
-                /* Find and create a new copy to link to */
-                let refLink = use.getAttribute("xlink:href");
-                let refElement = $(refLink)[0];
-                let replacement = $(refElement.outerHTML)[0];
-
-                /* Copy over any attributes on the link */
-                for (let i = 0; i < use.attributes.length; i++) {
-                    let attrib = use.attributes.item(i);
-                    if (attrib.name.toLowerCase() !== "xlink:href") {
-                        replacement.setAttribute(attrib.name, attrib.value);
+            try {
+                $(svg).find("use").each((_, use) => {
+                    /* Find and create a new copy to link to */
+                    let refLink = use.getAttribute("xlink:href");
+                    let refElement = $(refLink)[0];
+                    let replacement = $(refElement.outerHTML)[0];
+
+                    /* Copy over any attributes on the link */
+                    for (let i = 0; i < use.attributes.length; i++) {
+                        let attrib = use.attributes.item(i);
+                        if (attrib.name.toLowerCase() !== "xlink:href") {
+                            replacement.setAttribute(attrib.name, attrib.value);
+                        }
                     }
-                }
 
-                /* Replace the reference with the actual value */
-                use.parentNode.replaceChild(replacement, use);
-            });
+                    /* Replace the reference with the actual value */
+                    use.parentNode.replaceChild(replacement, use);
+                });
+            } catch (_err) {
+                /* If we're erroring, it's because the MathJax glyph cache hasn't loaded yet.
+                   We can retry again in a little bit to check if it's ready. */
+                setTimeout(callback, 100);
```

Comment:
I'm a bit worried that an actual error in the above code is going to result in an infinite loop and we'll never find out what happened. Perhaps we could do something like:
```suggestion
                if (retryCount++ < 100) {
                    setTimeout(callback, 100);
                } else {
                    throw err;
                }
```